gfx-examples/src/cube.rs

```diff
@@ -0,0 +1,310 @@
+extern crate cgmath;
+extern crate wgpu;
+
+#[path="framework.rs"]
```

Comment:
I think `path=` isn't commonly used so maybe we should just have `framework` or use an alias here